.github/workflows/pr.yml

```diff
@@ -1,42 +1,16 @@
-# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+# Copyright 2025, Proofcraft Pty Ltd
```

Comment:
Don't we need to keep the old copyright, or does this remove enough that it doesn't make sense now?

Either way, presumably we want to change the copyright for push.yml as well.

Reply:
There wasn't anything left of the old content, really, that's why I removed it. You are right that I should also do something in push.yml.